sematic/ui/packages/main/src/runs/RunIndex.tsx

```diff
@@ -176,6 +188,7 @@ export function RunIndex() {
                             label="Search"
                             variant="outlined"
                             onChange={onChange}
+                            defaultValue={defaultSearchString}
```

Comment:
We can use managed state by doing `value={searchString}`. We can set the default value for searchString to "" (empty string), or MUI will complain.